File: 백준/Silver/1764. 듣보잡/듣보잡.swift
// [Int] N, M (immutable)
// N: 듣도 못한 사람의 수 
// M: 보도 못한 사람의 수
// (범위(자연수): <= 20)
let arrayOfInput: Array<Int> = (readLine() ?? "1 1").split(separator: " ").compactMap { Int($0) }
let (N, M): (Int, Int) = (arrayOfInput[0], arrayOfInput[1])

// [Set<String>] setOfN (mutable): 듣도 못한 사람을 저장하는 집합 (길이: N)
var setOfN: Set<String> = Set<String>()

// [Set<String>] setOfM (mutable): 보도 못한 사람을 저장하는 집합 (길이: N)
var setOfM: Set<String> = Set<String>()

for _ in 0..<N {
    setOfN.insert( readLine() ?? "A" )
}
    
for _ in 0..<M {
    setOfM.insert( readLine() ?? "A" )
}
    
// [Set<String>] setOfIntersection (immutable): 듣도 보도 못한 사람을 저장하는 집합 (길이: <= N, M)
let setOfIntersection: Set<String> = setOfN.intersection(setOfM)

// 출력
print(setOfIntersection.count)
print(setOfIntersection.sorted().map { String($0) }.joined(separator: "\n"))

File: 백준/Silver/1764. 듣보잡/듣보잡.py
import sys

input = sys.stdin.readline

# [int] N, M
# N: 듣도 못한 사람의 수 
# M: 보도 못한 사람의 수
# (범위(자연수): <= 20)
N, M = map(int, input().split())

# [set(int)] setOfN: 듣도 못한 사람을 저장하는 집합 (길이: N)
setOfN = set()

# [set(int)] setOfM: 보도 못한 사람을 저장하는 집합 (길이: M)
setOfM = set()

for _ in range(N):
    setOfN.add(input().strip())
    
for _ in range(M):
    setOfM.add(input().strip())
    
# [set(int)] setOfIntersection: 듣도 보도 못한 사람을 저장하는 집합 (길이: <= N, M)
setOfIntersection = setOfN.intersection(setOfM)

# 출력
print(len(setOfIntersection))
print("\n".join(sorted(list(setOfIntersection))))